my-favorite-physicist.tex

```diff
@@ -60,5 +60,6 @@
 \input{content/instruction.tex}
 
 \input{content/albert_einstein.tex}
+\input{content/fermi.tex}
```

Comment:
mismatch with filenames ... either rename `content/Fermi.tex` to `content/fermi.tex`